### ISSUE TITLE
Remove jQuery dependency

### ISSUE THREAD

**Issue by clivebeckett** [clivebeckett/matomo-opt-out#1]
It shouldn’t be necessary to use jQuery in this script. It was just faster for me when writing it.

**Comment by Grienauer** [comment]
+1 from my side!
can we expect an update on this issue?
thx!

**Comment by clivebeckett** [comment]
Hi @Grienauer 

thanks for your note. I’m shocked this thing is two years old already :) At the moment I can’t do it as I am not very fluent in pure JS. I’d be happy to incorporate a Pull Request though :)

Cheers,
Clive.

**Comment by Grienauer** [comment]
hey! @clivebeckett I know that I had someone in my head who already fixed this. Finally I remembered and found him. Jon was so nice to "instantly" submit a Pull Request after I asked him :)
I love this kind of division of labour ;P

**Comment by clivebeckett** [comment]
Hi @Grienauer, hi @jonnitto,

sorry it had taken me so long to incorporate the PR. I was hesitant because it changed what should be a simple script to be an actual package. In my opinion this is completely unnecessary. However, I did merge the PR now and I’m happy this issue is being solved by it. In another step I removed all the files I deemed unnecessary, especially after Github told me about vulnerabilities within the dependencies added by @jonnitto.

Thanks a lot for your help!

Cheers,
Clive.
